app/src/main/res/values/colors.xml

```diff
@@ -4,4 +4,10 @@
   <color name="colorPrimaryDark">#00574B</color>
   <color name="colorAccent">#D81B60</color>
   <color name="ic_launcher_background">#26A69A</color>
+
+  <color name="light_brown">#AED68453</color>
```

Comment:
The name convention differs from the naming convention followed in `colorPrimaryDark`

I think is should be camelCase and therefore `lightBrown`